apps/backend/src/libs/modules/encrypt/base-encrypt.module.ts

```diff
@@ -0,0 +1,21 @@
+import bcrypt from "bcrypt";
+
+class BaseEncrypt {
+	constructor(public saltRounds: number) {}
```

Comment:
let's stick to one code style
```suggestion
  public saltRounds: number;
  
	constructor(saltRounds: number) {
	  this.saltRounds = saltRounds;
	}
```